api/scpca_portal/utils/helpers.py

```diff
@@ -16,30 +16,58 @@
 logger = get_and_configure_logger(__name__)
 
 
-def prep_data_dirs(wipe_input_dir: bool = False, wipe_output_dir: bool = True) -> None:
+def create_data_dirs(
```

Comment:
we can add a comment here to restrict writing to a directory that is outside of `settings.INPUT_DATA_DIR` or the output

Reply:
I've added validation to `remove_data_dirs` to prevent callers from passing directories outside the data directories. If an invalid directory is passed, an exception is raised. I've also updated the docstring in `create_data_dirs` to reflect this validation.

Additionally, I added a new `utils.validate_dir` function for directory validation. Please take a look!